deploy/helm/Makefile

```diff
@@ -41,9 +46,9 @@ helm_llama_stack_args = \
     $(if $(SAFETY_API_TOKEN),--set global.models.$(SAFETY).apiToken='$(SAFETY_API_TOKEN)',) \
     $(if $(LLAMA_STACK_ENV),--set-json llama-stack.secrets='$(LLAMA_STACK_ENV)',)
 
-helm_seed_admin_user_args = \
-    $(if $(ADMIN_USERNAME),--set seed.admin_user.username='$(ADMIN_USERNAME)',) \
-    $(if $(ADMIN_EMAIL),--set seed.admin_user.email='$(ADMIN_EMAIL)',)
+#helm_seed_admin_user_args = \
```

Comment:
Just wondering why these are commented out. I assume because we are not using them?  Wondering if we should delete instead?

Reply:
I left in place because I assumed we might need them once we enable llama-stack auth. I can remove them for now though.